AK/ByteBuffer.h

```diff
@@ -61,31 +61,32 @@ class ByteBuffer {
         return *this;
     }
 
-    [[nodiscard]] static Optional<ByteBuffer> create_uninitialized(size_t size)
+    [[nodiscard]] static ErrorOr<ByteBuffer> create_uninitialized(size_t size)
     {
         auto buffer = ByteBuffer();
-        if (buffer.try_resize(size).is_error())
-            return {};
-        return { move(buffer) };
+        TRY(buffer.try_resize(size));
+        return buffer;
```

Comment:
Some of these might end up copying, I'm not too sure right now
I'd suggest taking a look (maybe delete the copy ctor entirely to check), and replacing them with `{ move(buffer) }` where needed.